flow/model/signals.go

```diff
@@ -84,11 +84,6 @@ func (self TypedReceiveChannel[T]) ReceiveAsyncWithMoreFlag() (T, bool, bool) {
 	return result, ok, more
 }
 
-func (self TypedReceiveChannel[T]) Drain() {
```

Comment:
Removing this because:
1. after moving sync flow to activity it's not used
2. draining channels is generally a sign that you're doing something wrong; stuff like having to use this method is why I now avoid signals with temporal